scripts/reana

```diff
@@ -104,6 +106,91 @@ down () {
     kubectl delete -Rf configuration-manifests
 }
 
+demo () {
+    # Used to run REANA demo workflows.
+    # For now, hardcoded to run only helloworld-workflow.
+    demonstration=$2
+
+    # The code to check for flags is from http://stackoverflow.com/q/2875424
+    # Not sure if this is a good way or not.
+    flag_verbose=$(echo "$*" | grep -e "--verbose" -o)
+    flag_devel=$(echo "$*" | grep -e "--devel" -o)
```

Comment:
Maybe simple quotes to prevent command execution:

```
$ echo "$(date)"
Thu 27 Apr 18:40:02 CEST 2017
$ echo '$(date)'
$(date)
```

or rely on the flag being present last in $3 some place, or use environment variable...